scripts/CRLP_TEST_VALIDATE_ROLLUPS.cls

```diff
@@ -0,0 +1,696 @@
+/**
```

Comment:
@bethbrains Out of curiosity, why include this class in this PR as opposed to a new PR for it with the class and any related metadata files we want to related?

Reply:
It wasn't in this PR.... I committed it directly to feature/rollups, or at least I thought I did. Investigating now....

Reply:
Ahhh, I did merge to feature/rollups, but this branch got that because it's a child branch, and now we're merging this child branch into master directly. :/ Is that ok?

Reply:
Once we work this out, we should remove feature/rollups to avoid this confusion. We've been discussing this in our standup meetings to avoid losing all of our child branches getting merged to master. @force2b any objections?